plugins/image-prioritizer/tests/test-helper.php

```diff
@@ -553,6 +579,7 @@ static function () use ( $mobile_breakpoint, $tablet_breakpoint ): array {
 				',
 			),
 
+			// TODO: Eventually the images in this test should all be lazy-loaded, leaving the prioritization to the preload links.
```

Comment:
Not to be done in this PR. See https://github.com/WordPress/performance/pull/1261/files#r1625015100